WebExtension/data/inject/wrapper.js

```diff
@@ -1,7 +1,45 @@
 /* globals Readability */
 'use strict';
 
-var article = new Readability(document.cloneNode(true)).parse();
+// The implementation is from https://stackoverflow.com/a/5084441/260793
+function getSelectionHTML() {
+  var userSelection;
+  if (window.getSelection) {
```

Comment:
There is no need for this. All the supported browsers do support `window.getSelection()`

Reply:
okay, thank you, I will change and push later. :]